pkg/state/state_test.go

```diff
@@ -3455,3 +3455,19 @@ func TestAppendChartDownloadTLSFlags(t *testing.T) {
 		})
 	}
 }
+
+func TestHideChartURL(t *testing.T) {
+	tests := []struct {
+		input    string
+		expected string
+	}{
+		{"http://example.com@user", "http*****@user"},
```

Comment:
Normal url style should like this: http://username:password@example.com/. right? thanks so much. @NavesEdu

Reply:
Yes, I already change this in the test.